system-int-test/test/cmr/system_int_test/search/collection_spatial_keyword_search_test.clj

```diff
@@ -31,7 +31,7 @@
            "LA" [coll4]
```

Comment:
Can we add a test in here to test cmr-4192-iso-collection.xml with spatial search? So pretty much exactly what you did in the above file, but instead of keyword search do spatial-keyword search with the scenarios Kathy used when she wrote the ticket. I think the spatial test would really test what is in the ticket.